CMakePresets.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
This and the other JSON file should probably also be revered for now.

Reply:
no major objection on my side.